moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -53,21 +53,21 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
 {
   if (!planning_scene)
   {
-    ROS_ERROR_STREAM_NAMED("chomp_planner", "No planning scene initialized.");
+    ROS_ERROR_STREAM_NAMED("chomp_motion_planner", "No planning scene initialized.");
```

Comment:
I guess, `chomp` is sufficient.

Reply:
Not even that. The package name here is `chomp_motion_planner` already.
Unless chomp becomes so big that we need different subspaces I believe `ROS_ERROR_STREAM` is fine here.

Reply:
Well spotted.